src/prelude.rs

```diff
@@ -6,5 +6,4 @@
 
 pub use auto::traits::*;
 pub use editable_text::EditableTextExtManual;
-pub use glib::prelude::*;
```

Comment:
This line really don't needed?

Reply:
I don't see why we would? Otherwise we'll reexport all glib so better not.

Reply:
I thought that it was main idea: user need import only one prelude and get all traits in tree

Reply:
Won't it make conflicts of imports?

Reply:
IMHO Only if imported with direct name both time 😉

Reply:
Yeah this all sucks :) So we want to
- re-export the glib::prelude from the atk::prelude
- re-export the atk::prelude from atk:: but that includes the glib::prelude then

I think the above is not ideal but it will work. We should `#[doc(hide)]` the `glib::prelude` re-export in `atk::prelude` though, otherwise the docs will be awful.

There will be no conflicts from the compiler point of view as it's all the same types, just from different paths.

Reply:
So, keep this line and add `#[doc(hidden)]` above it. Don't delete the line

Reply:
I still don't get *why* we'd want to re-export anything that is not defined in the current crate.

Reply:
Because you'd expect to be able to use base class API without having to include the whole dependency tree. Just like in some random OOP language you'd expect to be able to use

```java
SourceView s = new SourceView(); // From GtkSourceView
s.show(); // from GtkWidget
```

Or do you want someone using sourceview to always include the preludes of glib, gio, gdk, gtk and sourceview instead of just the latter? :)

Reply:
I think what we want is
- `#[doc(hidden)] pub use glib::prelude::*` in both `crate::` and `crate::prelude`